client/src/modules/dashboard/Events/components/EventForm.tsx

```diff
@@ -20,6 +30,57 @@ import EventSponsorsForm from './EventSponsorsForm';
 import EventVenueForm from './EventVenueForm';
 import { EventFormProps, fields, EventFormData } from './EventFormUtils';
 
+interface IntegrationInfoData {
+  isAuthenticated: boolean | null | undefined;
+  hasCalendar: boolean;
+}
+
+const IntegrationInfo = ({
+  isAuthenticated,
+  hasCalendar,
+}: IntegrationInfoData) => {
+  const infoData = ({ isAuthenticated, hasCalendar }: IntegrationInfoData) => {
+    if (isAuthenticated) {
+      if (hasCalendar) {
+        return {
+          Icon: InfoIcon,
+          text: 'Instance is authenticated with calendar api, and chapter has created calendar. Chapter will attempt creating event in the calendar.',
+        };
```

Comment:
It doesn't seem necessary to warn people that things are going to work as expected

Reply:
I wanted to include it as a part if _something goes wrong_, as there isn't clear path to inform client when creating calendar event errors. 

This got me thinking, what if creating calendar and calendar event is pulled out of the create chapter/event resolvers? The actual creation can be then run once entry is created. That gives ability to separate and inform about errors on both these steps:

1. Create chapter/event. In case of errors, stop and inform client. Otherwise go to next step.
1. From client run mutation creating calendar/ calendar event. Errors can be clearly indicated as coming from calendar integration. They could be also displayed as _warnings_.

Reply:
I like the sound of that.

Potentially the Google Event creation could fail in a bunch of ways (lack of permissions, rate limits, network error, bad configuration etc. etc.), so rather than trying to warn organisers that the Event will not get created we should focus on handling the errors when it inevitably does fail. Particularly since there are no checks we can carry out that will guarantee that an Event will be created - only that it has been.

Even if the owner has forgotten to authenticate with Google, it should only be a problem for the first event. After which they'll see the warning, authenticate and create the calendar event (once that's implemented).

What do you reckon?

Reply:
That seems to be better way of handling it. I'll block this PR for now, start with simple informing if calendar/event in calendar was created, and see how to go from there.

Reply:
Cool, that sounds like the way to go.